ff/serving/provider/postgres.go

```diff
@@ -212,5 +245,28 @@ func (q postgresSQLQueries) numRows(n interface{}) (int64, error) {
 }
 
 func (q postgresSQLQueries) transformationCreate(name string, query string) string {
-	return fmt.Sprintf("CREATE TABLE %s AS %s ", sanitize(name), query)
+	return fmt.Sprintf("CREATE TABLE  %s AS %s", sanitize(name), query)
+}
+
+func (q postgresSQLQueries) transformationUpdate(db *sql.DB, tableName string, query string) error {
+	santizedName := sanitize(tableName)
+	tempName := sanitize(fmt.Sprintf("tmp_%s", tableName))
+	oldName := sanitize(fmt.Sprintf("old_%s", tableName))
+	transaction := fmt.Sprintf("BEGIN;"+
```

Comment:
This looks really similar to the transaction above, can we generalize?

Reply:
you got it

Reply:
Done